pkg/util/types/pvc.go

```diff
@@ -76,3 +77,15 @@ func IsSharedPVCFromClient(client kubecli.KubevirtClient, namespace string, clai
 	}
 	return
 }
+
+func IsPreallocated(annotations map[string]string) bool {
+	for a, value := range annotations {
+		if strings.Contains(a, "/storage.preallocation") && value == "true" {
```

Comment:
Should we search for the exact annotation (ie. cdi.kubevirt.io/storage.preallocation)?  Should this value be imported from cdi itself?